cmd/earthly/doc_cmds.go

```diff
@@ -0,0 +1,277 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/earthly/earthly/ast/spec"
+	"github.com/earthly/earthly/buildcontext"
+	"github.com/earthly/earthly/domain"
+	"github.com/earthly/earthly/earthfile2llb"
+	"github.com/earthly/earthly/features"
+	"github.com/earthly/earthly/util/platutil"
+	gwclient "github.com/moby/buildkit/frontend/gateway/client"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+)
+
+func (app *earthlyApp) actionDocumentTarget(cliCtx *cli.Context) error {
+	app.commandName = "docTarget"
+
+	if cliCtx.NArg() > 1 {
+		return errors.New("invalid number of arguments provided")
+	}
+
+	var tgtPath string
+	if cliCtx.NArg() > 0 {
+		tgtPath = cliCtx.Args().Get(0)
+		switch tgtPath[0] {
+		case '.', '/', '+':
+		default:
+			return errors.New("remote-paths are not currently supported - documentation targets must start with one of ['.', '/', '+']")
+		}
+	}
+
+	singleTgt := true
+	if !strings.ContainsRune(tgtPath, '+') {
+		tgtPath += "+base"
+		singleTgt = false
+	}
+
+	target, err := domain.ParseTarget(tgtPath)
+	if err != nil {
+		return errors.Errorf("unable to parse target [%v]", tgtPath)
```

Comment:
maybe use quotes instead -- we already do that in some places.
```suggestion
		return errors.Errorf("unable to parse target %q", tgtPath)
```